play/binary-explorer/css/style.css

```diff
@@ -23,6 +23,15 @@ button {
     text-align: center
 }
 
+#message-container {
+    text-align: center;
+    position: static;
```

Comment:
I don't think that ```position: static``` is necessary, and I'd rather text-align center be done with the [Bootstrap text-center helper class](https://getbootstrap.com/docs/4.4/utilities/text/#text-alignment). This is just to keep things consistent and keep the CSS file from getting too cluttered.